app/src/main/res/drawable/ic_room.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
```

Comment:
Please change the values to `13dp` to match the text size as defined in `@dimen/event_detailbar_text`. Sadly, we can not reference the resource item here because of the current minSdkVersion.

Reply:
I did the refactor in this commit
3337538